### DISCUSSION TITLE
Retry publish message if send failed

### DISCUSSION THREAD

**Discussion by idachour59** [drakkan/sftpgo#1427] [Q&A]
Good morning,

I am currently using sftpgo 2.5.3 to receive files and generate events to rabbitmq. I implemented eventrule and eventAction (HTTP call to send the event).

How can I simply perform Retrys if my RabbitMQ queue is no longer accessible?

Thank you for your feedback

**Comment by idachour59** [comment]
any answer guy ?

**Reply by idachour59** [reply]
Any reply friends ?

**Reply by Gmille80** [reply]
I'm interested too, any answer please ?
